src/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface Context {
```

Comment:
Can you add a brief description? Interested in:
1. Design consideration on why `AbortSignal` is in context and how you anticipate it is to be used/set.
2. Why the interface didn't implement/extend the Option interface?
3. How `author` and `attesters` will be set.

Reply:
Generally speaking, `Context` is a design pattern that is used to pass request-scoped values and cancellation signals across API boundaries. Context is common in node APIs and provided natively in [go](https://go.dev/blog/context)

1. `AbortSignal` is in context because it provides each caller up the stack to cancel any async operations. Here's a good [resource](https://blog.logrocket.com/complete-guide-abortcontroller-node-js/)
2. Which `Option` interface? The one provided via `interface-blockstore`?
3. My general thought was that both will be set through verifying auth and/or attestation

Reply:
Was more referring to the lack of documentation here for `Context` and I thought you were trying to be compatible with methods defined in `interface Blockstore extends Store<CID, Uint8Array>`, so thought was worthwhile adding some here.

Reply:
done: https://github.com/TBD54566975/dwn-sdk-js/commit/f474d19da9be9e9a1b11ca06b209705bba7a0148